kotlinx-coroutines-core/jvm/test/LCStressOptionsDefault.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package kotlinx.coroutines
+
+import org.jetbrains.kotlinx.lincheck.*
+import org.jetbrains.kotlinx.lincheck.strategy.stress.*
+import kotlin.reflect.*
+
+class LCStressOptionsDefault : StressOptions() {
+    init {
+        iterations(100 * stressTestMultiplierSqrt)
+        invocationsPerIteration(20_000 * stressTestMultiplierSqrt)
```

Comment:
This makes tests too slow on regular builds. See https://teamcity.jetbrains.com/viewLog.html?buildId=2571931&buildTypeId=KotlinTools_KotlinxCoroutines_BuildAggregated&tab=testsInfo
For example, `LockFreeListLCStressTest.testAddRemoveLinearizability` now takes 23 seconds.